sdk/core/abort-controller/review/abort-controller.api.md

```diff
@@ -9,7 +9,7 @@ export class AbortController {
     constructor(parentSignals?: AbortSignalLike[]);
     constructor(...parentSignals: AbortSignalLike[]);
     abort(): void;
-    readonly signal: AbortSignal;
+    get signal(): AbortSignal;
```

Comment:
this looks like it was generated by an old version of TS - 3.9 should mark this as readonly

Reply:
sorry for the irrelevant comment earlier, I deleted it. As for this issue, I will investigate further and get back to you.

Reply:
@xirzec okay, so it looks like there are no formatting changes required in this package. However, running `rushx extract-api` there resulted in this change. Also,
```
dalmahal@LAPTOP-FT6JHU6R:~/repo/sdk/core/abort-controller$ tsc -v
Version 3.9.5
```
Any idea what is going on here?

Reply:
Strange. If you do a `npm build` in this folder does it still do the same?

Reply:
no, `rushx build` does not do the same. The `build` script does not call the `extract-api` script.

Reply:
Is it sensible to revert back the result of executing the `extract-api` script in `abort-controller`? is `abort-controller` even maintained?

Reply:
Oh this package is maintained, it just hasn't updated much. I think reverting these changes for now is fine, but we should look into why extract-api is being buggy here.

Reply:
I see. I created https://github.com/Azure/azure-sdk-for-js/issues/10320. Please let me know if there are other issues in this PR.